packages/rainbowkit/src/wallets/walletConnectors/bitskiWallet/bitskiWallet.ts

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable sort-keys-fix/sort-keys-fix */
+import { InjectedConnector } from 'wagmi/connectors/injected';
+import { Chain } from '../../../components/RainbowKitProvider/RainbowKitChainContext';
+import { Wallet } from '../../Wallet';
+
+export interface BitskiWalletOptions {
+  chains: Chain[];
+  shimDisconnect?: boolean;
+}
+
+export const bitskiWallet = ({
+  chains,
+  shimDisconnect,
+}: BitskiWalletOptions): Wallet => {
+  const installed =
+    typeof window !== 'undefined' &&
+    typeof window.ethereum !== 'undefined' &&
+    ((window.ethereum as any).isBitski === true ||
+      window.ethereum.providers?.find((p: any) => p.isBitski === true));
+
+  return {
```

Comment:
The `=> ({ })` syntax visible in `braveWallet.ts` is preferred.

Reply:
@DanielSinclair - any updates here?